tests/kicker/test_basic_commands.py

```diff
@@ -0,0 +1,81 @@
+from parameterized import parameterized
+import unittest
+
+from utils.channel_access import ChannelAccess
+from utils.testing import parameterized_list
+from hamcrest import assert_that, is_, equal_to
+
+DEVICE_PREFIX = "KICKER_01"
+
+# VOLTAGE CALIBRATION CONSTANTS FOR TESTS
+DAQ_MAX_VOLTAGE = 10.0
+PSU_MAX_VOLTAGE = 45.0  # This is the default value set as a macro in IOC st.cmd
+VOLTAGE_CALIBRATION_RATIO = PSU_MAX_VOLTAGE / DAQ_MAX_VOLTAGE
+
+# CURRENT CALIBRATION CONSTANTS FOR TESTS
+DAQ_MAX_CURRENT = 10.0
+PSU_MAX_CURRENT = 15.0  # This is the default value set as a macro in IOC st.cmd
+CURRENT_CALIBRATION_RATIO = PSU_MAX_CURRENT / DAQ_MAX_CURRENT
+
+
+class BaseTests(unittest.TestCase):
+    record = None
+    calibration = None
+
+    def setUp(self):
+        self.ca = ChannelAccess(20, device_prefix=DEVICE_PREFIX)
+        self.ca.assert_that_pv_is("DISABLE", "COMMS ENABLED")
+        self._simulate_value(0)
+
+    def _simulate_value(self, value):
+        pv_root = "DAQ:{}".format(self.record)
```

Comment:
This method depends on the state that the class is in, which makes reading/debugging challenging. Could the data held in `self.record` be passed in as an argument instead?